packages/tools/src/stateManagement/segmentation/polySeg/Labelmap/convertContourToLabelmap.ts

```diff
@@ -256,14 +256,42 @@ function _getAnnotationMapFromSegmentation(
     ? options.segmentIndices
     : Array.from(annotationMap.keys());
 
-  const annotationUIDsInSegmentMap = new Map<number, Annotation[]>();
+  const annotationUIDsInSegmentMap = new Map<number, any>();
   segmentIndices.forEach((index) => {
     const annotationUIDsInSegment = annotationMap.get(index);
 
-    const annotations = Array.from(annotationUIDsInSegment).map((uid) => {
-      const annotation = getAnnotation(uid);
+    // Todo: there is a bug right now where the annotationUIDsInSegment has both
+    // children and parent annotations, so we need to filter out the parent
```

Comment:
Should I make the changes in `contourSegmentationCompleted` to remove it?

Reply:
Yes later